bioio/tests/test_array_like_reader.py

```diff
@@ -1139,3 +1139,9 @@ def test_aicsimage_from_array(
             dict,
         ),
     )
+
+
```

Comment:
@toloudis is this sort of what you had in mind for testing the type offset?

Reply:
Yep!  passing file path into arraylikereader raises exception.  LGTM!